ats/jobs/models.py

```diff
@@ -0,0 +1,40 @@
+from django.db import models
+from model_utils import Choices
+from django.utils.translation import ugettext_lazy as _
+
+
+class Category(models.Model):
```

Comment:
What do you think to move this model in a standalone app? This model may be shared across multiple apps

Reply:
Actually, I can't imagine how this`Category` model can be shared between models for now so, if you have an example please provide me.